src/components/WorkOrder/TasksAndSors/TasksAndSorsView.js

```diff
@@ -44,7 +46,11 @@ const TasksAndSorsView = ({ workOrderReference, tabName }) => {
       ) : (
         <>
           {tasksAndSors && (
-            <TasksAndSorsTable tasksAndSors={tasksAndSors} tabName={tabName} />
+            <TasksAndSorsTable
+              tasksAndSors={tasksAndSors}
```

Comment:
Not a big deal but I wonder if it's worth giving this property and variable a more specific name now that we have `originalTasksAndSors` to avoid potential confusion later. Does it make sense for this to be `postVariationTasksAndSors` perhaps?

Reply:
I think this can look exactly the same as `originalTasksAndSors` if no variation was done with it. How about `currentTasksAndSors` or something to refer to it being the latest snapshot of the live tasks and sors against the work order